coalib/parsing/ConfParser.py

```diff
@@ -156,7 +157,12 @@ def __parse_lines(self, lines, origin):
                                     remove_empty_iter_elements=
                                     self.__remove_empty_iter_elements),
                                     # Stop ignoring
-                            allow_appending=(keys == []))
+                            allow_appending=(
+                                keys == [] or keys == [('', 'ignore')]))
+        for key in current_section.contents:
+            if(key == 'ignore'):
+                current_section.contents[key].value =\
```

Comment:
Use of `\` for line break should not be used.

Reply:
Do you want me to put brackets around it?

Reply:
@sangamcse is (correctly) telling you what needs to be done.  You then need to find out how to achieve it.  "line continuations" , i.e. `\`, are a tokenizer level escape sequence to *pretend* the EOL doesnt exist.  It is done before the structure of the file is analysed.  Most parsers break in unusual ways because of them.  line positions are often wrong, because EOLs that dont exist ... are hard to count.

The easiest approach is to use round brackets instead.

```py
current_section.contents[key].value = (
    current_section.contents[key].value.replace('\n', ',')
```

but that is still ugly as.

The problem is the rhs is veryyyy long.

Also you are using `current_section.contents[key]` several times.

Find ways to make it easier for someone else to read.